health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/FacilityPopup.js

```
@@ -358,8 +323,8 @@ const FacilityPopUp = ({ details, onClose }) => {
           onClose={onClose}
           heading={`${t(`MICROPLAN_ASSIGNMENT_FACILITY`)} ${details?.additionalDetails?.facilityName}`}
           children={[
-            <div>
-              <div className="card-container" style={{ border: "1px solid #D6D5D4", borderRadius: "3px" }}>
+            <div className="facilitypopup-serach-results-wrapper">
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)